ext/storage_sources/s3_store/s3_log_system.h

```diff
@@ -27,5 +26,5 @@ class s3_log_system: public Aws::Utils::Logging::LogSystemInterface {
     void LogVerboseMessage(
       const char *tag, Aws::Utils::Logging::LogLevel logLevel, const std::string &message);
     std::atomic<Aws::Utils::Logging::LogLevel> logLevel;
-    WT_EXTENSION_API *wt_api;
+    WT_EXTENSION_API *wtApi;
```

Comment:
np: Should we use the prefix `m_` for class members?

Reply:
No, we are not planning to do that.